CONTRIBUTING.md

```diff
@@ -1,5 +1,10 @@
 ## Development environment
 
+It's recommended to use `uv` for setting up your development environment, but
+this is not required. `uv` is available [as a snap](https://snapcraft.io/astral-uv)
+and from [official sources](https://docs.astral.sh/uv/getting-started/installation/).
+If using `uv`, substitute `virtualenv` with `uv venv` and `pip` with `uv pip`.
```

Comment:
if `uv` is recommended, these instructions should be along the lines of _if not using ..._

Reply:
Easier to just not explain how to use something else anyway :-)